src/BackTube.js

```diff
@@ -156,12 +156,14 @@ export default class BackTube {
   onPlayerStateChange(state: Object) {
     const { playerSettings } = this.options;
     if (playerSettings) {
-      const { start = 0, loop } = playerSettings;
+      const { start, loop } = playerSettings;
+      const seek = start != null ? start : 0;
```

Comment:
Making sure even if start was `null` value, it will pass `0` value to `player.seekTo`.